### PR TITLE
To do list: add js functionality buttons

### PR DESCRIPTION
**Pull request by alexiscyber14**
Requirements:
Remove all hardcoded items from the tasks array.
Create a new JavaScript file for the new functionality.
Implement a function for adding a new task (add a new element to the array).
Implement a function for deleting a task (remove an element from the array).
Implement a function for editing task descriptions.
All changes to the To Do List should be saved in local storage.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,42 +1,128 @@
...
+  itemsArray.push(item.value);
+  localStorage.setItem('items', JSON.stringify(itemsArray));
+  window.location.reload();
+}
+
+document.querySelector('#enter').addEventListener('click', () => {
+  const item = document.querySelector('#item');
+  createItem(item);
+});
+
```
Comment (ridwanediallo) [inline src/index.js:72]: - [ ] You did an excellent job with adding and removing a task 👍️. However, there are a few things that should be worked on in order to meet all requirements. By default, new tasks should have a completed property set to false and a index property set to the new array length, and when a task is deleted, the index is updated. Please make an effort to do so.👌😊
### Currently what we have is this 👇️

Screenshot from 2023-02-22 13-07-17

### What we should have is something like this 👇️
```
{task: "go", index: 1, completed: false}
```

**Review by ridwanediallo — CHANGES_REQUESTED** [review]
# Changes Required ♻️
## Hi @alexiscyber14, 
AlmostThereReachGIF

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- Your PR has a good title and a descriptive summary. ✔


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (alexiscyber14) [reply src/index.js:72]: Okay thank you. lemme try fix it

**Review by rotimiazeez — APPROVED** [review]
Hi @alexiscyber14 👏🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
You documented your work professionally. 👏🏽 
GitHub flow is used correctly. 👍🏽 
Your PR has a good title and a descriptive summary. 👍🏽 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me(@rotimiazeez) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**